tests/test_lambdas/test_broker.py

```diff
@@ -0,0 +1,40 @@
+from unittest.mock import MagicMock
+
+import pytest
+
+from lbz.lambdas.broker import LambdaBroker
+
+
+class TestEventBroker:
+    def test_broker_works_properly(self) -> None:
+        func_1 = MagicMock(return_value="some_data")
+        mapper = {"x": func_1}
+        event = {"op": "x", "data": {"y": 1}}
+
+        resp = LambdaBroker(mapper, event).react()  # type: ignore
+
+        assert resp == "some_data"
+        func_1.assert_called_once_with({"y": 1})
+
+    def test_broker_responds_raises_implemented_when_event_type_is_not_recognized(self) -> None:
```

Comment:
[#981509875](https://github.com/pdyba/lambdalizator/pull/64#discussion_r981509875):
```suggestion
    def test_broker_raises_error_when_event_type_is_not_recognized(self) -> None:
```